docs/src/explicit/SSPRK.md

```diff
@@ -0,0 +1,32 @@
+# OrdinaryDiffEqSSPRK
+
+Solvers specialized on certain PDE types.
```

Comment:
SSPRK methods are Runge-Kutta methods which support the "strongly preserving property" (SSP). They are
designed for the use in discretizations of hyperbolic partial differential equations and conservation laws
as they have extra stability properties ( e.g., stability with respect to total variation, the maximum norm, or other convex functionals) when step-size restrictions are respected. In particular, these properties are granted if the step-size is kept to a level where the CFL coefficients are less than the SSP coefficient.

Note that for SSPRK methods, a algorithm utility `OrdinaryDiffEqCore.ssp_coefficient(alg)` exists that allows for querying the SSP coefficient for use in step size calculations.